.circleci/config.yml

```diff
@@ -46,3 +47,29 @@ jobs:
           path: rd-cli-base/build/test-results
       - store_test_results:
           path: rd-api-client/build/test-results
+
+  snyk_test:
+    docker:
+      - image: cimg/openjdk:11.0 # this should match what's used in the rest of the project
+    steps:
+      - checkout
+      - setup_remote_docker
+      - git/rebase_on_main
```

Comment:
hmm, do we need to rebase on main? I see the PD git orb states its purpose is to prevent "unwanted merge checks from overwriting existing deployed code", but I don't think that applies here

Reply:
We probably could. This was in the original template that SRE-Delivery worked with me on, but it's possible it's not relevant here. I'll take that out in the next iteration.